src/webgl/p5.RendererGL.js

```diff
@@ -468,6 +473,17 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     this.spotLightAngle = [];
     this.spotLightConc = [];
 
+    // Null if no image light is set, otherwise, it is set to a p5.Image
+    // made a blurrytexture attribute to be used in imageLight function
+    // this is to lookup image from blurrytexture Map rather from a
+    // texture map
+    this.diffusedTextures = new Map();
+    // map to store the created textures to prevent mis calculation
+    this.specularTextures = new Map();
+    // property to be set true after imageLight is called
```

Comment:
Slight update to this: we don't set it to true, we set it to a `p5.Image`. Instead of checking if it's true, we check if it's not null